### PR TITLE
zh-CN: translate error-handling

### PR DESCRIPTION
**Pull request by mgeisler**
Part of #324.

### CODE REVIEW

File: po/zh-CN.po [hunk 1]
```diff
@@ -10284,17 +10294,25 @@ msgid ""
...
-msgstr ""
+msgstr "# 使用 `?` 传播错误"
 
 #: src/error-handling/try-operator.md:3
 msgid ""
 "The try-operator `?` is used to return errors to the caller. It lets you "
 "turn\n"
 "the common"
-msgstr ""
+msgstr "try 操作符 `?` 用于将错误返回给调用方。它能把常用命令\n"
```
Comment (wnghl) [inline po/zh-CN.po:10315]:
```suggestion
msgstr ""
"try 操作符 `?` 用于将错误返回给调用方。它能把常用命令\n"
```
to fix the「Test / build (zh-CN) (pull_request) 」error

File: po/zh-CN.po [hunk 2]
```diff
@@ -10166,25 +10166,27 @@ msgstr ""
...
 "* Functions that can have errors list this in their return type,\n"
 "* There are no exceptions."
 msgstr ""
+"* 包含错误的函数会在返回类型中列出相关信息。\n"
+"* 此规则没有例外。"
 
 #: src/error-handling/panics.md:1
 msgid "# Panics"
-msgstr ""
+msgstr "# 紧急警报"
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/zh-CN.po:10185]: I think we could! Let's ping some other Chinese translators such as @superwhd and @Doris-Ge to see what they think. Perhaps they can help align the translation afterwards.

[the remark this answers is not in the document]
Comment (superwhd) [reply po/zh-CN.po:10185]: I agree on keeping `panic` untranslated.

[the remark this answers is not in the document]
Comment (mgeisler) [reply po/zh-CN.po:10185]: Me too, by the way :slightly_smiling_face:

```suggestion
msgstr "# Panics"
```

Let me know if there are other small things we should fix before merging this.

**Comment by mgeisler** [comment]
Hi @wnghl, the suggestions have been applied — are you happy to merge this now?

File: po/zh-CN.po [hunk 3]
```diff
@@ -10166,25 +10166,27 @@ msgstr ""
...
 
 #: src/error-handling/panics.md:1
 msgid "# Panics"
-msgstr ""
+msgstr "# Panics"
 
 #: src/error-handling/panics.md:3
 msgid "Rust will trigger a panic if a fatal error happens at runtime:"
-msgstr ""
+msgstr "如果运行时发生严重错误，Rust 会触发紧急警报："
```
Comment (wnghl) [inline po/zh-CN.po:10189]:
```suggestion
msgstr "如果运行时发生严重错误，Rust 会触发 panic："
```

File: po/zh-CN.po [hunk 4]
```diff
@@ -10202,16 +10204,19 @@ msgid ""
 "  * Panics are symptoms of bugs in the program.\n"
 "* Use non-panicking APIs (such as `Vec::get`) if crashing is not acceptable."
 msgstr ""
+"* 紧急警报用于指示不可恢复的意外错误。\n"
+"  * 紧急警报反映了程序中的 bug 问题。\n"
+"* 如果崩溃不可接受，请使用不会触发紧急警报的 API（例如 `Vec::get`）。"
```
Comment (wnghl) [inline po/zh-CN.po:10209]:
```suggestion
"* Panic 用于指示不可恢复的意外错误。\n"
"  * Panic反映了程序中的 bug 问题。\n"
"* 如果崩溃不可接受，请使用不会触发 panic 的 API（例如 `Vec::get`）。"
```

File: po/zh-CN.po [hunk 5]
```diff
@@ -10202,16 +10204,19 @@ msgid ""
...
 msgid "# Catching the Stack Unwinding"
-msgstr ""
+msgstr "# 捕获堆栈展开"
 
 #: src/error-handling/panic-unwind.md:3
 msgid ""
 "By default, a panic will cause the stack to unwind. The unwinding can be "
 "caught:"
-msgstr ""
+msgstr "默认情况下，紧急警报会导致堆栈展开。您可以捕获展开信息："
```
Comment (wnghl) [inline po/zh-CN.po:10219]:
```suggestion
msgstr "默认情况下，panic 会导致堆栈展开。您可以捕获展开信息："
```

**Review by wnghl — COMMENTED** [review]
Hi @mgeisler, It's fine to merge this. I have kept the term 'panic' untranslated in other places and I'll approve this.

**Review by wnghl — APPROVED** [review]

**Comment by mgeisler** [comment]
> Hi @mgeisler, It's fine to merge this. I have kept the term 'panic' untranslated in other places and I'll approve this.

Thanks @wnghl!

**Pull request merged**